pkgs/applications/virtualization/singularity/generic.nix

```diff
@@ -14,8 +14,8 @@
 }:
 
 let
-  # Workaround for vendor-related attributes not overridable (#86349)
-  # should be removed when the issue is resolved
+  # Backward compatibility layer for the obsolete workaround of #86349
```

Comment:
Can we keep the "vendor-related attributes not overridable" stil?

Reply:
Addressed.